src/components-examples/cdk-experimental/scrollable-table-body/cdk-scrollable-table-body-flex/cdk-scrollable-table-body-flex-example.html

```diff
@@ -0,0 +1,33 @@
+<button mat-button color="primary" (click)="addRow()">Add row</button>
+<button mat-button color="primary" (click)="removeRow()">Remove row</button>
+<button mat-button color="primary" (click)="nextMaxHeight()">Change max height</button>
```

Comment:
nit: I don't think you're supposed to have multiple `primary` buttons next to each other.

Reply:
Ah, I did that out of habit. My team's UX mocks tend to have `primary` buttons next to each other.

I updated the template to align with the `table` demo (raised, default buttons).